dev-tools/src/main/resources/ant/integration-tests.xml

```diff
@@ -157,6 +157,14 @@
       <attribute name="jvm.args" default="${tests.jvm.argline}"/>
       <element name="additional-args" optional="true"/>
     <sequential>
+      <!-- make sure no elasticseatch instance is currently running and listening on the port we need -->
+      <fail message="This test expects port @{es.http.port} to be free but an elasticsearch instance already running and listening on port @{es.http.port}.
```

Comment:
Why not?

"This test expects port @{es.http.port} to be free but an elasticsearch instance is already running and listening on that port."